src/applications/mhv/medical-records/components/RecordList/RecordList.jsx

```diff
@@ -30,6 +38,15 @@ const RecordList = props => {
     return [from, to];
   };
 
+  useEffect(
+    () => {
+      return () => {
+        dispatch(resetPagination(location.pathname));
```

Comment:
Use the `useLocation` hook to get `location` here. See how we are doing it in Navigtion.jsx. As it is, is is throwing a linter warning: "Unexpected use of 'location'. eslint (no-restricted-globals)"